src/reducers/index.js

```diff
@@ -3,19 +3,18 @@ import workspaceReducer from './workspace';
 import windowsReducer from './windows';
 import manifestsReducer from './manifests';
 import configReducer from './config';
-import infoResponsesReducer from './infoResponses';
```

Comment:
I think this maybe snuck in with the repo transfer. This removes the infoResponses stuff I added.